app/component/map/tile-layer/DynamicParkingLots.js

```diff
@@ -59,6 +59,19 @@ class DynamicParkingLots {
       );
     });
 
+  getIcon = type => {
+    if (type === 'Parkhaus') {
+      return 'covered_carpark';
+    }
+    if (type === 'Park-Ride') {
```

Comment:
p+m should be included as well (situated at A81/B296)

Reply:
But is that in this layer?

Reply:
It should, as it was in the static son before.

Reply:
![Screenshot from 2020-02-27 09-51-53](https://user-images.githubusercontent.com/151346/75427476-d8512200-5946-11ea-9faa-e960d52b3de3.png)

Sorry, it just isn't but I can add it. Let me know if this isn't the correct place.